content/data-sharing/mutable-and-sendable.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Mutable and Sendable"
+section: "Data Sharing"
+menu:
+  toc:
+    parent: "data-sharing"
+    weight: 40
+---
+
+## Problem
+
+Suppose you have a central Supervisor actor that manages a very large data structure that contains the work produced by a collection of Worker actors.  However, the Superviser must also respond to data requests (perhaps from the Workers themselves, or in this case, from another set of Requester actors).  As a result, the Superviser's data structure must be both mutable (to update it in response to feedback from Workers) and sendable (to respond to requests from Requesters).  `iso` is the only refcap that accommodates that, but our Supervisor can't use `iso` here because it must keep a reference to the data structure to continue to respond to incoming updates and requests.  How to proceed?
```

Comment:
" `iso` is the only refcap that accommodates that"

what "that" refers to is unclear.

Reply:
I think you have a really good introduction to the problem here, but you should let it breathe. This is a rather complicated scenario and I think you should look to expand the explanation of the problem quite a lot. Like 3-4x more content to dig in more seems completely reasonable (if not more).

I don't think I would understand the problem were it not for working through a solution with you in Zulip.